reframe/REFRAME_KNOWN_METHODS.md

```diff
@@ -269,7 +269,7 @@ A message for indicating that the client is able to act as a provider for a give
 
 ```ipldsch
     type ProvideRequest struct 
-        Key &Any
+        Key [&Any]

```

Comment:
LGTM. This could be a kinded union for 1 or more, but the couple extra bytes for a single provide don't seem like a big deal.